source/features/table-input.tsx

```diff
@@ -100,10 +83,7 @@ function add(anchor: HTMLElement): void {
 }
 
 function init(signal: AbortSignal): void {
-	observe([
-		'md-ref', // TODO: Drop in June 2024, cleanup button JSX above too
```

Comment:
Sorry, early removal. Given the slew of changes, we can't really afford to support GHE everywhere